meerk40t/gui/scene/scene.py

```diff
@@ -1384,7 +1386,7 @@ def event(self, window_pos=None, space_pos=None, event_type=None):
             return RESPONSE_CONSUME
         elif event_type == "lost":
             return RESPONSE_CONSUME
-        elif str(event_type).startswith("zoom"):
+        elif str(event_type).startswith("zoom "):
```

Comment:
That space is needed since the zoom-in and zoom-out would fit. The question was is the first active word there actually zoom the hyphen forms are to avoid catching this since these are different zoom styles.